dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java

```diff
@@ -124,6 +124,9 @@ public AnnotationCollection getAnnotations() {
 	 * @return a {@link SequenceNode} with the information of the given {@link Node}.
 	 */
 	public EnrichedSequenceNode createSequenceNode(Node node) {
+		if (node == null) {
```

Comment:
This check does not belong here, this method should never be passed a null value...